### PR TITLE
proposed updates to Contributing

### PR DESCRIPTION
**Pull request by jlovejoy**
proposed updates to Contributing to reflect discussions on Nov 1st call and use of new Submit License Tool

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,39 +5,42 @@ Please familiarize yourself with the SPDX License List and its supporting docume
...
-* Provide a short explanation regarding the need for this license or exception to be included on the SPDX License List, and identify at least one program that uses it.
-* Provide the text of the license. 
-
-### Review Process
-1. The SPDX Legal Team will discuss any submissions for new licenses or exceptions on the bi-weekly calls. If further information or clarification is required, the Team will reach out to the requestor accordingly.
-2. Issues will be labled either "new license/exception: Accepted" or "new license/exception: Not Accepted" as appropriate with an explanation and the Issue closed for the latter case.
-3. Upon acceptance, the new license or exception will be prepared by a member of the SPDX Legal Team in the proper XML format and plain text via a Pull Request and reviewed as appropriate. 
+3. Submit your request via the [SPDX Online Tool, Submit New License] (http://13.57.134.254/app/submit_new_license/) using the following guidance, also see [overview-fields] (https://spdx.org/spdx-license-list/license-list-overview#fields) for more info on some of these fields. Note, you must have a Github id in order to use this tool/process.
```
Comment (swinslow) [inline CONTRIBUTING.md]:
```suggestion
3. Submit your request via the [SPDX Online Tool, Submit New License](http://13.57.134.254/app/submit_new_license/) using the following guidance, also see [overview-fields](https://spdx.org/spdx-license-list/license-list-overview#fields) for more info on some of these fields. Note, you must have a Github id in order to use this tool/process. 
```

Comment (swinslow) [reply CONTRIBUTING.md]: Very minor change -- I think removing the spaces between the square brackets and parentheses may be necessary in order to make the Markdown links appear correctly.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -5,39 +5,42 @@ Please familiarize yourself with the SPDX License List and its supporting docume
...
-2. Issues will be labled either "new license/exception: Accepted" or "new license/exception: Not Accepted" as appropriate with an explanation and the Issue closed for the latter case.
-3. Upon acceptance, the new license or exception will be prepared by a member of the SPDX Legal Team in the proper XML format and plain text via a Pull Request and reviewed as appropriate. 
+3. Submit your request via the [SPDX Online Tool, Submit New License] (http://13.57.134.254/app/submit_new_license/) using the following guidance, also see [overview-fields] (https://spdx.org/spdx-license-list/license-list-overview#fields) for more info on some of these fields. Note, you must have a Github id in order to use this tool/process. 
+
+* Full Name: Provide a proposed Full Name for the license or exception. 
+* Short Identifier: Provide a proposed Short Identifier. 
+* Source/URL Provide a functioning URL reference to the license or exception text, preferably from the author or a community recognized source.
```
Comment (swinslow) [inline CONTRIBUTING.md]:
```suggestion
* Source/URL: Provide a functioning URL reference to the license or exception text, preferably from the author or a community recognized source.
```

Comment (swinslow) [reply CONTRIBUTING.md]: nit edit, adding colon to align with other items in the list

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -5,39 +5,42 @@ Please familiarize yourself with the SPDX License List and its supporting docume
...
+* Text: Full text of the license or exception. 
+* Comments: Provide a short explanation regarding the need for this license or exception to be included on the SPDX License List, and identify at least one program that uses it.
+
+4. Follow the discussion or any request for additional information via the Issue and comments. Note, if the SPDX-legal team has questions that go unanswered for several months, we will close the issue without making a final decision. 
+
+## Review Process
+
+1. The SPDX Legal Team will discuss any submissions for new licenses or exceptions via comments in the Github issue. Please follow the comments and respond accordingly if there are questions or additional information requested.
+2. New licenses may be approved if 3 SPDX-legal team members (2 lawyers, 1 non-lawyer) sign-off that the license is acceptable AND there is no objection raised from the greater SPDX-legal community. If there are objections, then the issue will be labelled "discuss on legal call" and will be discussed on an upcoming bi-weekly call.
```
Comment (swinslow) [inline CONTRIBUTING.md]:
```suggestion
2. New licenses may be approved if 3 SPDX-legal team members (2 lawyers, 1 non-lawyer) sign-off that the license is acceptable AND there is no objection raised from the greater SPDX-legal community within the Github issue comments. If there are objections, then the issue will be labelled "discuss on legal call" and will be discussed on an upcoming bi-weekly call. 
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -5,39 +5,42 @@ Please familiarize yourself with the SPDX License List and its supporting docume
...
+* Comments: Provide a short explanation regarding the need for this license or exception to be included on the SPDX License List, and identify at least one program that uses it.
+
+4. Follow the discussion or any request for additional information via the Issue and comments. Note, if the SPDX-legal team has questions that go unanswered for several months, we will close the issue without making a final decision. 
+
+## Review Process
+
+1. The SPDX Legal Team will discuss any submissions for new licenses or exceptions via comments in the Github issue. Please follow the comments and respond accordingly if there are questions or additional information requested.
+2. New licenses may be approved if 3 SPDX-legal team members (2 lawyers, 1 non-lawyer) sign-off that the license is acceptable AND there is no objection raised from the greater SPDX-legal community. If there are objections, then the issue will be labelled "discuss on legal call" and will be discussed on an upcoming bi-weekly call. 
+2. Issues will be lablled either "new license/exception: Accepted" or "new license/exception: Not Accepted" as appropriate with an explanation and the Issue closed for the latter case.
```
Comment (swinslow) [inline CONTRIBUTING.md]:
```suggestion
3. Issues will be labelled either "new license/exception: Accepted" or "new license/exception: Not Accepted" as appropriate with an explanation and the Issue closed for the latter case.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -5,39 +5,42 @@ Please familiarize yourself with the SPDX License List and its supporting docume
...
+3. If accepted, the new license or exception will be prepared by a member of the SPDX Legal Team in the proper XML format and plain text via a Pull Request and reviewed as appropriate.
```
Comment (swinslow) [inline CONTRIBUTING.md]:
```suggestion
4. If accepted, the new license or exception will be prepared by a member of the SPDX Legal Team in the proper XML format and plain text test file via a Pull Request and reviewed as appropriate. 
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -5,39 +5,42 @@ Please familiarize yourself with the SPDX License List and its supporting docume
...
+
+## Review Process
+
+1. The SPDX Legal Team will discuss any submissions for new licenses or exceptions via comments in the Github issue. Please follow the comments and respond accordingly if there are questions or additional information requested.
+2. New licenses may be approved if 3 SPDX-legal team members (2 lawyers, 1 non-lawyer) sign-off that the license is acceptable AND there is no objection raised from the greater SPDX-legal community. If there are objections, then the issue will be labelled "discuss on legal call" and will be discussed on an upcoming bi-weekly call. 
+2. Issues will be lablled either "new license/exception: Accepted" or "new license/exception: Not Accepted" as appropriate with an explanation and the Issue closed for the latter case.
+3. If accepted, the new license or exception will be prepared by a member of the SPDX Legal Team in the proper XML format and plain text via a Pull Request and reviewed as appropriate. 
 4. The new license/exception will be officially added for the next release of the SPDX License List.
```
Comment (swinslow) [inline CONTRIBUTING.md]:
```suggestion
5. The new license/exception will be officially added for the next release of the SPDX License List.
```

**Comment by swinslow** [comment]
Looks good to me  =)

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -5,39 +5,42 @@ Please familiarize yourself with the SPDX License List and its supporting docume
...
-You can join the mailing list and otherwise manage your subscription [via spdx-legal mailman](https://lists.spdx.org/mailman/listinfo/spdx-legal). Information on the bi-weekly calls and the working area for the SPDX Legal Team, can be found on the [SPDX legal team wiki](https://wiki.spdx.org/view/Legal_Team)
+The SPDX License List is maintained by the SPDX Legal Team. Work and discussion is primarily done via:
+* mailing list: our traditional form of communication. Meeting reminders, minutes, updates are posted there, as well as discussion.
+* this Github repo: comments and issues and PRs related to specific changes to the files that comprise the SPDX License List, e.g., update a URL, recommend additional markup for matching purposes. In particular, new license requests are tracked here.
+* bi-weekly calls: used to discuss topics and issues that may be difficult to only discuss via email or via Github.
+You can join the mailing list and otherwise manage your subscription [via spdx-legal mailman](https://lists.spdx.org/mailman/listinfo/spdx-legal).
```
Comment (waldyrious) [inline CONTRIBUTING.md]: I think it would make more sense to make this line part of the "mailing list" bullet point.

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -5,39 +5,42 @@ Please familiarize yourself with the SPDX License List and its supporting docume
...
+The SPDX License List is maintained by the SPDX Legal Team. Work and discussion is primarily done via:
+* mailing list: our traditional form of communication. Meeting reminders, minutes, updates are posted there, as well as discussion.
+* this Github repo: comments and issues and PRs related to specific changes to the files that comprise the SPDX License List, e.g., update a URL, recommend additional markup for matching purposes. In particular, new license requests are tracked here.
+* bi-weekly calls: used to discuss topics and issues that may be difficult to only discuss via email or via Github.
+You can join the mailing list and otherwise manage your subscription [via spdx-legal mailman](https://lists.spdx.org/mailman/listinfo/spdx-legal). 
+Information on the bi-weekly calls and the working area for the SPDX Legal Team, can be found on the [SPDX legal team wiki](https://wiki.spdx.org/view/Legal_Team)
```
Comment (waldyrious) [inline CONTRIBUTING.md]: This line could be merged into the "bi-weekly calls" entry on the list above.

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -5,39 +5,42 @@ Please familiarize yourself with the SPDX License List and its supporting docume
...
-4. The new license/exception will be officially added for the next release of the SPDX License List.
-
-## Other Contributions to the SPDX License List
-If you find an error or have a suggested update to an existing license, please open a [new issue](https://github.com/spdx/license-list-XML/issues/new). Depending on the nature of the Issue, it may be referred for further discussion on the mailing list or bi-weekly calls. 
-Examples of things that should be logged via an issue (versus raised on the mailing list/discussed on calls) may include:
-* update an URL for a license
-* recommend additional markup for matching purposes (usually requires legal review)
+3. Submit your request via the [SPDX Online Tool, Submit New License](http://13.57.134.254/app/submit_new_license/) using the following guidance, also see [overview-fields](https://spdx.org/spdx-license-list/license-list-overview#fields) for more info on some of these fields. Note, you must have a Github id in order to use this tool/process.
```
Comment (waldyrious) [inline CONTRIBUTING.md]:
```suggestion
3. Submit your request via the SPDX Online Tool [Submit New License](http://13.57.134.254/app/submit_new_license/) using the following guidance. See also [overview-fields](https://spdx.org/spdx-license-list/license-list-overview#fields) for more info on some of these fields. Note: you must have a Github account in order to use this tool/process. 
```

File: CONTRIBUTING.md (hunk 7 above)
Comment (jlovejoy) [reply CONTRIBUTING.md]: good idea, just changed

File: CONTRIBUTING.md (hunk 8 above)
Comment (jlovejoy) [reply CONTRIBUTING.md]: done

**Pull request merged**